### PR TITLE
[App] Placement and Property: remove unused includes

### PR DESCRIPTION
**Pull request by donovaly**

### CODE REVIEW

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_6435">pipeline status</a> for feature branch [PR_6435](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_6435). Pipeline [477358449](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/477358449) was triggered at [6ab3ba0](https://github.com/FreeCAD/FreeCAD/pull/6435/commits/6ab3ba060a24615901526101445e089b7d4f56d6). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_6435">pipeline status</a> for feature branch [PR_6435](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_6435). Pipeline [477358934](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/477358934) was triggered at [d9b9402](https://github.com/FreeCAD/FreeCAD/pull/6435/commits/d9b9402a62e6c7a2f69ec93e9579bb632d3299ec). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

File: src/App/Placement.h [hunk 1]
```diff
@@ -27,18 +27,12 @@
 #include <Base/Placement.h>
 
 #include "FeaturePython.h"
-#include "GeoFeature.h"
```
Comment (wwmayer) [inline src/App/Placement.h:30]: Placement inherits from App::GeoFeature

File: src/App/Property.cpp [hunk 2]
```diff
@@ -23,20 +23,14 @@
 
 #include "PreCompiled.h"
 
-#ifndef _PreComp_
-#	include <cassert>
```
Comment (wwmayer) [inline src/App/Property.cpp:27]: assert() is used in some functions

File: src/App/PropertyContainerPyImp.cpp [hunk 3]
```diff
@@ -23,17 +23,10 @@
 
 #include "PreCompiled.h"
 
-#ifndef _PreComp_
-# include <sstream>
```
Comment (wwmayer) [inline src/App/PropertyContainerPyImp.cpp:27]: std::stringstream is used

File: src/App/PropertyContainerPyImp.cpp [hunk 4]
```diff
@@ -23,17 +23,10 @@
...
-#endif
-
 #include "PropertyContainer.h"
 #include "Property.h"
-#include "PropertyLinks.h"
-#include "Application.h"
 #include "DocumentObject.h"
 
-#include <boost/iostreams/device/array.hpp>
```
Comment (wwmayer) [inline src/App/PropertyContainerPyImp.cpp:36]: basic_array_source is used

File: src/App/PropertyExpressionEngine.h [hunk 5]
```diff
@@ -23,12 +23,9 @@
 #ifndef EXPRESSIONENGINE_H
 #define EXPRESSIONENGINE_H
 
-#include <boost/unordered/unordered_map.hpp>
 #include <boost/function.hpp>
-#include <boost_signals2.hpp>
```
Comment (wwmayer) [inline src/App/PropertyExpressionEngine.h:28]: boost::signals2::signal is used

File: src/App/PropertyExpressionEngine.h [hunk 6]
```diff
@@ -23,12 +23,9 @@
 #ifndef EXPRESSIONENGINE_H
 #define EXPRESSIONENGINE_H
 
-#include <boost/unordered/unordered_map.hpp>
```
Comment (wwmayer) [inline src/App/PropertyExpressionEngine.h:26]: boost::unordered_map is used

File: src/App/PropertyStandard.h [hunk 7]
```diff
@@ -24,21 +24,18 @@
 #ifndef APP_PROPERTYSTANDARD_H
 #define APP_PROPERTYSTANDARD_H
 
-// Std. configurations
-
-
-#include <memory>
```
Comment (wwmayer) [inline src/App/PropertyStandard.h:30]: std::shared_ptr is used

File: src/App/PropertyUnits.cpp [hunk 8]
```diff
@@ -23,26 +23,13 @@
 
 #include "PreCompiled.h"
 
-#ifndef _PreComp_
-# include <boost/version.hpp>
-# include <boost/filesystem/path.hpp>
-# include <cfloat>
```
Comment (wwmayer) [inline src/App/PropertyUnits.cpp:29]: DBL_MAX is used

**Comment by wwmayer** [comment]
c7910825a3c8154

**Pull request closed without merging**